<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="proper_widget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>491</width>
      <height>401</height>
     </rect>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>481</width>
       <height>311</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="formulas_widget"/>
    </widget>
    <widget class="QPushButton" name="select_eq_btn">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>340</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Дальше</string>
     </property>
    </widget>
    <widget class="QComboBox" name="select_eq_comboBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>340</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="holder">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>1</y>
      <width>821</width>
      <height>601</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="solve_widget" native="true">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>681</width>
      <height>521</height>
     </rect>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>-1</y>
       <width>681</width>
       <height>451</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="solve_v_layout"/>
    </widget>
    <widget class="QPushButton" name="solve_btn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>470</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Решить</string>
     </property>
    </widget>
    <widget class="QLabel" name="res_label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>470</y>
       <width>511</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>x=</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="chooseTaskWidget" native="true">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>69</y>
      <width>551</width>
      <height>391</height>
     </rect>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>-1</y>
       <width>551</width>
       <height>391</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="goPropBtn">
        <property name="text">
         <string>Собственный интеграл</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="goInpropBtn">
        <property name="text">
         <string>Несобственный интеграл</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="inproper_widget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>491</width>
      <height>401</height>
     </rect>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>481</width>
       <height>311</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="inproper_widget_2"/>
    </widget>
    <widget class="QPushButton" name="select_inprop_btn">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>340</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Дальше</string>
     </property>
    </widget>
    <widget class="QComboBox" name="select_inprop_comboBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>340</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="solve_inprop_widget" native="true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>681</width>
      <height>521</height>
     </rect>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_6">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>-1</y>
       <width>681</width>
       <height>451</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="solve_inprop_layout"/>
    </widget>
    <widget class="QPushButton" name="solve_inprop_btn">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>470</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Решить</string>
     </property>
    </widget>
    <widget class="QLabel" name="inprop_res_label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>470</y>
       <width>511</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <zorder>holder</zorder>
   <zorder>inproper_widget</zorder>
   <zorder>chooseTaskWidget</zorder>
   <zorder>solve_widget</zorder>
   <zorder>proper_widget</zorder>
   <zorder>solve_inprop_widget</zorder>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
